qcodes/instrument_drivers/Lakeshore/lakeshore_base.py

```diff
@@ -266,15 +273,18 @@ def wait_until_set_point_reached(self,
                                    self.wait_equilibration_time.get_latest())
 
         active_channel_id = self.input_channel()
-        active_channel_number_in_list = active_channel_id - 1
-        active_channel = \
-            self.root_instrument.channels[active_channel_number_in_list]
+        active_channel = getattr(
+            self.root_instrument,
+            (self.root_instrument
```

Comment:
For readability I would prefer making this its own variable outside the getattr. channel_name or some such thing

Reply:
Done.